site-packages/integralstor_utils/django_utils.py

```diff
@@ -0,0 +1,43 @@
+
+def in_request_parameters(request, param_name):
```

Comment:
I first wrote this thinking it may be the default one being used but then realized that we actually needed the value in most cases from either get or post. 

Regarding the function name, the reason for the s at the end is that it checks all the paramS in request (either get or post depending on the request type), to see if the param_name is present. Can change the name. Let me know what you think.